tests/Form/InputSearchTest.php

```
@@ -0,0 +1,620 @@
+        self::assertSame(
+            <<<HTML
+            <input id="inputsearch-" type="search" form="form-id">
+            HTML,
+            InputSearch::tag()->form('form-id')->id('inputsearch-')->render(),
+            "Failed asserting that element renders correctly with 'form' attribute.",
+        );
+    }
+
+    public function testRenderWithGlobalDefaultsAreApplied(): void
+    {
+        SimpleFactory::setDefaults(InputSearch::class, ['class' => 'default-class']);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\UIAwesome\Html\Core\Factory\SimpleFactory' in method 'testRenderWithGlobalDefaultsAreApplied'. (undefined)

(StaticAccess)